### PR TITLE
create login form fix #38

### LINKED ISSUE #38: Create twitter login page
Time to create an important page in this project, login page

https://twitter.com/login

Screenshot 2021-09-11 at 11 32 41 PM

very important to focus as well on the behaviors, check that page to understand the requirements on focus and colors

### PR DESCRIPTION
**Pull request by mahaelbasuony**
Screenshot 2021-09-13 at 08 14 59

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
+/*---------------------------------------------->> GLOBAL - START  <<<----------------------------------------------*/
+:root {
+  --main-bg-color: #151f2b;
+  --text-color: #ffffff;
+  --input-text-bg-color: #253341;
+  --primary-font: Arial, Helvetica, sans-serif;
+  --tweet-button-color: #1a91da;
+  --gray-800: #8394A1;
```
Comment (medhatdawoud) [inline login.css:8]: Nice making variables from the beginning 💪

File: login.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+}
+.page {
+  /* margin: 0;
+  padding: 0; */
```
Comment (medhatdawoud) [inline login.css]: why this is commented out?

File: login.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+.login-contianer__userid, .login-contianer__password {
+  position:relative; 
+  margin-bottom:25px; 
+  width:100%;
+  
+  
+}
+
+.userid__input, .password__input  {
```
Comment (medhatdawoud) [inline login.css]: better to give one generic form-control class and use it everywhere, imagine having to use more controls, will you add more selectors here?

File: login.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  position:absolute;
+  pointer-events:none;
+  left:10px;
+  top:20px;
+  transition:0.2s ease all; 
+  -moz-transition:0.2s ease all; 
+  -webkit-transition:0.2s ease all;
+}
+
+.userid__input:focus ~ label, .userid__input:not(:placeholder-shown) ~ label, .password__input:focus ~ label, .password__input:not(:placeholder-shown) ~ label{
```
Comment (medhatdawoud) [inline login.css]: Save here, please add one generic class and put all styles into it

File: login.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  padding-top: 10px;
+}
+.userid__input:focus, .password__input:focus{
+  font-size: 16px;
+  border: none;
+  border: 2px solid var(----tweet-button-color);
+  
+
+}
+label {
```
Comment (medhatdawoud) [inline login.css]: Add a class name to label and use it here instead, don't use element names for styling

File: login.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+  color: var(--gray-800);
+  
+  position:absolute;
+  pointer-events:none;
```
Comment (medhatdawoud) [inline login.css:71]: could you explain to me here in a comment what does this property do ?

File: login.css [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+.actions__forgot-password:hover, .actions__sign-in:hover{
+  
+  text-decoration: underline;
+}
+.actions__forgot-password::after{
+  content: ".";
+  color: var(--gray-800);
+  padding-left: 4px;
+}
```
Comment (medhatdawoud) [inline login.css:115]: Good one 😉

File: login.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+</head>
+<body class="page">
+  <div class="page__inner" >
+    <div class="login-contianer">
+      <div class="login-contianer__logo">
+        <i class="fa fa-twitter"></i>
+      </div>
+      <div class="login-contianer__title">Log in to Twitter</div>
+      <div class="login-contianer__userid">
+        <input class="userid__input" type="text" placeholder=" ">
```
Comment (medhatdawoud) [inline login.html]: NO need to add `placeholder` attribute if you are not going to use it

**Review by medhatdawoud — CHANGES_REQUESTED** [review]

File: login.css [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+  font-size: 16px;
+  /* width:calc(100% - 12px); */
+  width:100%;
+  height:60px;
+  border: 2px solid var(--input-text-bg-color);
+  background-color: transparent;
+  color: var(--text-color);
+  border-radius: 5px;
+  padding-left: 10px ;
+  padding-top: 10px;
```
Comment (medhatdawoud) [inline login.css]: padding should be symetric, right = left and top = bottom

**Review by medhatdawoud — CHANGES_REQUESTED** [review]
overall good job, one more thing to be fixed

File: login.css (hunk 9 above)
Comment (mahaelbasuony) [reply login.css]: why?

Comment (mahaelbasuony) [reply login.css]: i tried it but didn't get same result

**Review by medhatdawoud — APPROVED** [review]

**Pull request merged**